main.py

```diff
@@ -1,10 +1,61 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players:
+        players_data = json.load(players)
+    for player_name in players_data:
+        Player.objects.create(
+            nickname=player_name,
+            email=players_data[player_name]["email"],
+            bio=players_data[player_name]["bio"],
+            race=player_race_check_and_create(
+                players_data[player_name]["race"]
+            ),
+            guild=player_guild_check_and_create(
+                players_data[player_name]["guild"]
+            )
+        )
+
+
+def player_race_check_and_create(race: dict) -> Race:
+    if not Race.objects.filter(
+            name=race["name"]
+    ).exists():
+        Race.objects.create(
+            name=race["name"],
+            description=race["description"]
+        )
+    race_skills_check_and_create(race)
+    return Race.objects.get(name=race["name"])
```

Comment:
try to use get_or_create method to simplify ur code
the same is for similar cases